rfc/0011-value-transformer/README.md

```diff
@@ -0,0 +1,191 @@
+      "gesetzliche Ziele" with PercentTransformer
+    ];
+}
+```
+
+This version only works with transformers that have exactly one input and exactly one output. We might need to come up with a different mechanism to allow other transformers (requires future RFCs).
+
+
+### Usage 2: TableTransformerBlock
+There is a newly introduced `TableTransformerBlock` block type that applies the mapper to a column in a `table`.
+
+Example:
+```
+block GermanToBooleanBlock oftype TableTransformerBlock {
+  inputColumns: ["Kritisch"];  // array to satisfy all inputs of transformer
+  outputColumns: ["IsCritical"]; // might also overwrite one of the input columns
+  use: GermanToBooleanTransformer;
+}
+```
+
+### Usage 3: SheetTransformerBlock
+There is a newly introduced `SheetTransformerBlock` block type that applies the mapper to a set of cells in a `sheet`. Please note that the value type is not stored in sheets and thus lost after computation. Outputs are therefore parsed to string automatically by the block.
+
+Example:
+```
+block GermanToBooleanBlock oftype SheetTransformerBlock {
+  inputCells: [range A3:A17]; // if multiple arguments: have to match in dimensions
+  outputCells: [range D3:D17]; // has to match dimension of inputs
+  use: GermanToBooleanTransformer;
+}
+```
+
+
+## Drawbacks
+- declaration of in and outputs might be much boilerplate.
+- TableInterpreter blocks only allow transformers with one input and one output.
+
+## Alternatives
+
+- use other keyword, e.g., `vtrans`, `mapper`, ...
+- shorten `input` and `output` keywords to `in` and `out`
+- Transformers cannot work on sheets
+- design nicer syntax for multiple inputs and outputs via names instead of relying on the index
+- refactor TableInterpreter completely to also allow transformers with multiple in and outputs
+
+## Possible Future Changes/Enhancements
+
+- We could introduce a pipe-like syntax that allows chaining transformers.
+
+### Variables in Transformers
+Storing interim results can improve coding experience.
+```
+transformer <name> {
+  // inputs and outputs
+  var <varBane> oftype <valuetype>: <expression producing valuetype>; // alternative: infer type automatically 
+  // use variable to compute outputs
+}
+```
+
+### Composite ValueTypes
+
+Composite ValueTypes could be served with this implementation by extending the assignment statement:
+```
+// "MyStreet" and "7b" => composite valuetype
+transformer AddressComposer {
+  input streetName oftype text;
+  input houseNumber oftype text;
+  output address oftype Address; // has properties "streetName" oftype StreetName and "houseNumber" oftype HouseNumber 
+
+  // "." notation to write on property of composite value type
+  address.streetName: streetName using StreetNameReader; // StreetNameReader is other transformation text -> StreetName
+  address.houseNumber: houseNumber using HouseNumberReader; // HouseNumberReader is other transformation text -> HouseNumber
+}
+```
+
+### Mappers
```

Comment:
This has to be called `match on` I beg you.

Reply:
[789fba5](https://github.com/jvalue/jayvee/pull/283/commits/789fba58202bbf7fef9967816122a4c5b932804f)